PlutoHelperAgentTests/GetRequiredVersionTests.m

```diff
@@ -0,0 +1,51 @@
+//
+//  GetRequiredVersionTests.m
+//  PlutoHelperAgentTests
+//
+//  Created by David Allison on 07/04/2022.
+//  Copyright © 2022 Guardian News & Media. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "GetRequiredVersion.h"
+
+@interface GetRequiredVersionTests : XCTestCase
+
+@end
+
+@implementation GetRequiredVersionTests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+}
+
+- (void)testGetRequiredVersion {
+    NSArray *testArray = [[NSArray alloc] initWithObjects:@"2.0.0",@"1.0.0",nil];
+    NSString *outputVersion = [GetRequiredVersion getRequiredVersion:testArray];
```

Comment:
this implies to me that `getRequiredVersion` should actually be called `getLatestVersion`?

Reply:
I called it this because the URL parameter it is used to populate is called 'requiredVersion'. I will change its name if you like.

Reply:
Done.